src/main.ts

```diff
@@ -10,13 +10,17 @@ import { SqlOpsDataClient, ClientOptions } from 'dataprotocol-client';
 import { IConfig, ServerProvider, Events } from 'service-downloader';
 import { ServerOptions, TransportKind } from 'vscode-languageclient';
 
+import registerCommands from './commands';
 import * as Constants from './constants';
 import ContextProvider from './contextProvider';
 import * as Utils from './utils';
 import { Telemetry, LanguageClientErrorHandler } from './telemetry';
+import { CommandObserver } from './CommandObserver';
 
 const baseConfig = require('./config.json');
+const packageJson = require('../package.json');
 const outputChannel = vscode.window.createOutputChannel(Constants.serviceName);
+const projectOutputChannel = vscode.window.createOutputChannel(Constants.projectOutputChannel);
```

Comment:
Should we only create it when command is being ran? I don't see any reason to have it listed there if you don't use projects/never build. Or is it not going to show up until you write to it?

Reply:
It will show up as soon as it is created.

Reply:
Then we probably should encapsulate it in our own output class, where we can check if it's already created or create it and make commands use that one instead.

Reply:
yeah doing that.